course/sitemaps.py

```diff
@@ -1,10 +1,26 @@
+import datetime
+
 from django.contrib import sitemaps
 from django.urls.base import reverse
 from django.utils import timezone
 
 from .models import CourseInstance, CourseModule
 
 
+DELTA_DAYS_NEW = 180
+
+
+def is_new(item):
+    now = timezone.now()
+    delta = datetime.timedelta(DELTA_DAYS_NEW)
+    is_new = (
```

Comment:
This is probably just my stupidity, but the reasoning for is_new logic did not completely open up to me. Perhaps a small inline comment would be in place to provide reasoning in this thinking?